ansible/roles/nginx/templates/rogeros_auth_proxy.conf.j2

```diff
@@ -22,30 +22,30 @@ server {
         location /chronos/scripts/views/job_detail_header_view.js {
 
             auth_basic Off;
-            root /usr/local/nginx/chronos/auth;
+            root /usr/local/openresty/nginx/chronos/auth;
         }
 
         location /chronos/scripts/templates/job_persistence_error.hbs {
 
             auth_basic Off;
-            root /usr/local/nginx/chronos/auth;
+            root /usr/local/openresty/nginx/chronos/auth;
         }
 
         location /chronos/scripts/views/job_detail_view.js {
 
             auth_basic Off;
-            root /usr/local/nginx/chronos/auth;
+            root /usr/local/openresty/nginx/chronos/auth;
         }
 
 
         location /marathon/ {
-		# goes to another method if basic http auth fails
-		# in our case, it will always go to the aaadaemon
-		# because http credential files is empty
-		satisfy all;
-		auth_basic "Restricted";
-		auth_basic_user_file {{nginx_conf_dir}}/{{nginx_basic_auth_file_name}};
-		auth_request /auth-proxy;
+                # goes to another method if basic http auth fails
+                # in our case, it will always go to the aaadaemon
+                # because http credential files is empty
```

Comment:
The above 3 comment lines appears to be no longer valid (unrelated to this story.) Best to remove them if so.